config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
+    resources :users
     get 'static_pages/home'
     get 'static_pages/help'
```

Comment:
Dùng nháy kép trong các file rb, erb e nhé
```suggestion
    get "static_pages/home"
    get "static_pages/help"
```